scripts/start.sh

```diff
@@ -7,8 +7,18 @@ set -e
 ScriptDir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 cd "$ScriptDir"
 
+# get the port from the REACT_APP_BACKEND_URI env variable
+envContent=$(grep -v '^#' ../webapp/.env | xargs)
+backendPort=$(echo $envContent | sed -n 's/.*:\([0-9]*\).*/\1/p')
+
 # Start backend (in background)
 ./start-backend.sh &
 
+# check that the backend is running and keep checking until it is
+while ! nc -z localhost $backendPort; do
```

Comment:
Please add the retry logic to this script too

Reply:
If you look at the start.sh file diff I did implement the retry logic.  Am I missing something?

Reply:
Looks better now except you've removed the .gitignore from the repo!

Please push it back in and this PR should be good to go.

Reply:
Yep, sorry about that.